packages/api/lambdas/index-from-database.js

```diff
@@ -159,6 +159,7 @@ async function indexModel({
       let translationResult;
       try {
         translationResult = await translationFunction(pageResult);
+        await esClient.refreshClient();
```

Comment:
It would be nice if this refresh behavior was only used in the EsClient model rather than lambdas needing to handle it. I don't have a good idea for how to do that though. I see why we're here. It's not feasible to introduce an AWS call for credentials with every single invocation of a method on EsClient.

Is there anything we could do to be _less_ proactive? If we were to wait until there's an error and _then_ refresh in the model? 

Thinking out loud, mostly just trying to get to where you ended up 😄

Reply:
Sure - let's chat re: this

Reply:
Did chat. Same as https://github.com/nasa/cumulus/pull/3627#discussion_r1572428366, we could definitely do this but it's a question of time/effort vs. this narrowly scoped problem and the knowledge that ES isn't here to stay.